pkgs/applications/version-management/git-and-tools/git-crypt/default.nix

```diff
@@ -12,16 +14,26 @@ stdenv.mkDerivation rec {
     inherit name;
   };
 
+  nativeBuildInputs = [ libxslt ];
+
   buildInputs = [ openssl makeWrapper ];
 
   patchPhase = ''
     substituteInPlace commands.cpp \
       --replace '(escape_shell_arg(our_exe_path()))' '= "git-crypt"'
   '';
 
-  installPhase = ''
-    make install PREFIX=$out
-    wrapProgram $out/bin/* --prefix PATH : $out/bin:${git}/bin:${gnupg}/bin
+  makeFlags = [
+    "ENABLE_MAN=yes"
+    "DOCBOOK_XSL=${docbook_xsl}/share/xml/docbook-xsl-nons/manpages/docbook.xsl"
+  ];
+
+  installFlags = [
+    "PREFIX=${placeholder ''out''}"
```

Comment:
I think it might have been my mistake to make this an `installFlags`? (should be `makeFlags`)

Reply:
Isn't it only used by `make install`?

Reply:
Generally this is an unneeded specificity.

Reply:
Okay! 😄